src/lib/test.js

```diff
@@ -150,6 +150,26 @@ const Mixin = function() {
         return dotObjectKey.get(this.context, key)
       }.bind(this),
     },
+    $prompt: {
```

Comment:
Since it is an external API I think we need to put some safety checks in to ensure the API exists and if it doesn’t we continue gracefully. 

I’d check if prompt exists and if the API is there, if not we can do two things:
A> Continue gracefully as the Storm consumer does not have the prompt API exposed
B> Fail the test forcefully. This ensures the tester doesn’t get false positives because the invoker doesn’t support the API requested by the test during execution.

I’m leaning towards B, what do you think?

Reply:
Wouter, 

I am planning to add the below to cover B. Is that fine? 
selectChoices: (msg, choice, waitTime) => {
          if(this.prompt && typeof this.prompt.selectChoices === 'function')
            return this.prompt.selectChoices(msg, choice, waitTime)
          else {
            throw new Error('Prompt not supported')
          }
      }

Reply:
Yep that looks good to me.